README.md

```diff
@@ -23,18 +26,25 @@ Paste the following Terraform sample module into file clickhouse-eks.tf in a new
 ```hcl
 provider "aws" {
   # https://registry.terraform.io/providers/hashicorp/aws/latest/docs
+  # region     = "us-east-1"
+  # access_key = "my-access-key"
+  # secret_key = "my-secret-key"
 }
 
-module "eks_clickhouse" {
-  source  = "github.com/Altinity/terraform-aws-eks-clickhouse"
+variable "region" {
+  default     = "us-east-1"
+}
+
+module "aws_eks_clickhouse" {
+  source  = "Altinity/eks-clickhouse/aws"
```

Comment:
Lets' keep git source from now, I will avoid publishing to TF registry until this is finished